From d33afd1290e5c92861936c0d26a25383a21d70d4 Mon Sep 17 00:00:00 2001
From: Alan Viverette <alanv@google.com>
Date: Fri, 2 Jun 2017 10:37:32 -0400
Subject: [PATCH 14/67] Move <permission> tags to correct parent element

AAPT2 is stricter about enforcing correctness, and the permission tags
were in the wrong place.

Fixes: 62262214
Test: builds with AAPT2
Change-Id: I20fb827f81d9b57404c6c02bdfa27acf3e572c90
---
 AndroidManifest.xml | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 25371f922..3eba78a81 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -249,12 +249,6 @@
             </intent-filter>
         </activity>
 
-        <permission android:name="com.android.gallery3d.filtershow.permission.READ"
-                    android:protectionLevel="signature" />
-
-        <permission android:name="com.android.gallery3d.filtershow.permission.WRITE"
-                    android:protectionLevel="signature" />
-
         <provider
             android:name="com.android.gallery3d.filtershow.provider.SharedImageProvider"
             android:authorities="com.android.gallery3d.filtershow.provider.SharedImageProvider"
-- 
2.17.1

